content/en/apps/guides/performance/telemetry.md

```diff
@@ -48,18 +49,27 @@ The telemetry data gathered changes with different versions of the framework. Cu
 | `boot_time:purging_failed` | The purging failed when running on device startup. Added in 3.14. |
 | `boot_time:purging_meta:<boolean>` | `boot_time:purging_meta:true` when purging of the local meta database ran successfully, `boot_time:purging_meta:false` when it did not run. Added in 3.14. |
 | `boot_time:purging_meta_failed` | The purging of the local meta database failed. Added in 3.14. |
-| `enketo:reports:<form>:<action>:<component>` | The time taken to fill in Enketo forms. The `action` can either be "add" or "edit". The `component` is one of: "render" covers getting the form and rendering it on screen; "user_edit_time" is the time the user took to fill in and submit the form; or "save" is about converting the form into a report and saving it. |
-| `enketo:contacts:<form>:<action>:<component>` | As above but for Contact forms. The `action` can either be "add" or "edit". |
+| `contact_list:load` | The time taken to load contact list. Added in 4.6. |
+| `contact_list:query` | The time taken to query contacts. This includes free text search and sort. Added in 4.6.  |
```

Comment:
> It's not clear to me if this is talking only about the search bar on the contacts tab or if this is a subset of contact_list:load

It's both. When the contact list loads (`contact_list:load`) it does a query (`contact_list:query`) to bring the data. And when the CHW searches by a term or do a sort, the same query process (`contact_list:query`) is recorded. ([The code](https://github.com/medic/cht-core/blob/900218e9b4594c11e01809d1f1333cd955e7d310/webapp/src/ts/modules/contacts/contacts.component.ts#L313), for the technical eyes)

> What are all the different user interactions that will record against (ie count towards) this telemetry key?

The initial load of Contact List when visiting the tab, search and sort. They are all the same process. The duration recorded includes:
- Build the search criteria 
- Search (here it records `search:contacts:<filter[:filter]>`)
- Once the search has finished, update the home place 
- Format the search results so it can be displayed - includes calculation of visits and overdue if enabled.
- Update the scroll in the view 
- Update FAB
- Send for display
After all this, the query process is considered finished and the Apdex is calculated (last line of the function).

Not sure how to explain all this in the docs 😅

Reply:
I managed to add a bit of detail to make it too long, I definitely need feedback about how to improve it